src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/discussions/PostsListPage.java

```diff
@@ -9,12 +9,15 @@
 
 import java.util.List;
 
-public class PostsListPage extends WikiBasePageObject {
+/**
+ * @ownership Social Wikia
```

Comment:
remove `@ownership` tag, according to instructions we received on recent qm tech meeting